src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -671,6 +670,7 @@ export const BorrowInput = ({
                   quoteErrorTranslation ||
                   isEstimatedFeesDataError ||
                   isEstimatedFeesDataLoading ||
+                  isAddressByteCodeLoading ||
```

Comment:
move to `isLoading`

Reply:
done in [02b8f00](https://github.com/shapeshift/web/pull/7358/commits/02b8f00e75f94ff8231733ed9f5ed61b8627c0d6)